### PR TITLE
Alex

### PR DESCRIPTION
**Pull request by AlexJDuncan**
Functionality:
- FavCompany button now adds and deletes companies to/from smoves[i].favCompanies on database

gitignore:
- had package-lock.json listed 2 times.  I just removed 1.

CSS:
- Made a couple changes to sizes on carousel.  probably not final changes to those.

Note:
- I have not put in accessibility.  that is my next goal.  needed to get this up for the Task List team

### CODE REVIEW

File: client/components/LocalCompanies/BussinessCard.jsx [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-  height: 30rem;
+  height: 550px;
   margin: 15px;
 `;
 
 const BussinessCard = ({ data }) => {
   //* * States * *//
   const [buttonColor, setButtonColor] = useState('secondary');
+  const { userData, setUserData } = useContext(UserDataContext);
+  console.log(userData.smoves.filter(smove => smove.isCurrentSmove)[0].favCompanies.filter(company => company.companyName !== data.name));
```
Comment (danitian) [inline client/components/LocalCompanies/BussinessCard.jsx]: did you mean to leave in this console log?

File: client/components/LocalCompanies/BussinessCard.jsx [hunk 2]
```diff
@@ -15,6 +17,8 @@ const CardStyle = styled.div`
...
```
Comment (johnkwak08) [inline client/components/LocalCompanies/BussinessCard.jsx]: do you need this console.log?

File: client/components/LocalCompanies/BussinessCard.jsx (hunk 1 above)
Comment (greta-schock) [inline client/components/LocalCompanies/BussinessCard.jsx]: rogue console.log

**Review by johnkwak08 — CHANGES_REQUESTED** [review]

**Review by greta-schock — CHANGES_REQUESTED** [review]

**Review by danitian — COMMENTED** [review]
lgtm!

**Review by KilgoreTrout9 — APPROVED** [review]
remove console.log and add more comments

File: client/components/LocalCompanies/BussinessCard.jsx (hunk 2 above)
Comment (AlexJDuncan) [reply client/components/LocalCompanies/BussinessCard.jsx]: fixed it

File: client/components/LocalCompanies/BussinessCard.jsx (hunk 1 above)
Comment (AlexJDuncan) [reply client/components/LocalCompanies/BussinessCard.jsx]: fixed it

Comment (AlexJDuncan) [reply client/components/LocalCompanies/BussinessCard.jsx]: fixed it

**Pull request merged**